### ISSUE TITLE
Problem having different column name than property name in v3.4.0

### ISSUE THREAD

**Issue by aserrabi** [borisdj/EFCore.BulkExtensions#491]
Hi,

There's an issue while calling the method BulkUpdateAsync 

If the entity model has a column name different than the property name it throws an `Invalid operation exception: PropertyName 'MyPropName' specified in 'PropertiesToInclude' not found in Properties.`

Steps to reproduce:

1. Create a model like this:
```
public class MyEntityModel
{
  [Column("my_prop_name")]
  public long MyPropName { get; set; }
} 
```

2. Call the following method with 'MyPropName' property name in the PropertiesToInclude array.
```
context.BulkUpdateAsync(
        entities,
        new BulkConfig { PropertiesToInclude = propertiesToInclude });
```

**Comment by borisdj** [comment]
Should be now fixed with 3.4.2

**Comment by aserrabi** [comment]
Thanks a lot, it's working with 3.4.2
